### PR TITLE
feat: add security classification view FS-95

### PR DESCRIPTION
**Pull request by sb88k**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/FS-95" title="FS-95" target="_blank">StoryFS-95</a>  [iOS] [TOP] Indicate classified status for users and conversations
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

Add security classification view for using in user profile and conversation screens.

### Dependencies

Needs releases with:

- [ ] https://github.com/wireapp/wire-ios-data-model/pull/1334
- [ ] https://github.com/wireapp/wire-ios-request-strategy/pull/482
- [ ] https://github.com/wireapp/wire-ios-sync-engine/pull/1661
- [ ] https://github.com/wireapp/wire-ios-sync-engine/pull/1664
- [ ] https://github.com/wireapp/wire-ios/pull/5544
- [ ] https://github.com/wireapp/wire-ios/pull/5545
- [ ] https://github.com/wireapp/wire-ios/pull/5547

### CODE REVIEW

**Review by johnxnguyen — APPROVED** [review]
Shall we merge to a feature branch so we can collect changes before merging to develop?

File: Wire-iOS/Sources/UserInterface/UserProfile/Views/SecurityLevelView.swift [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+    func configure(with classification: SecurityClassification) {
+        securityLevelLabel.font = FontSpec(.small, .bold).font
+
+        switch classification {
+        case .none:
+            isHidden = true
+
+        case .classified:
+            securityLevelLabel.text = "Classified" // TODO: Translation: Need to clarify
```
Comment (David-Henner) [inline Wire-iOS/Sources/UserInterface/UserProfile/Views/SecurityLevelView.swift:46]: even though there's no translations, you could already localise this

File: Wire-iOS/Sources/UserInterface/UserProfile/Views/SecurityLevelView.swift [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+            securityLevelLabel.textColor = UIColor.from(scheme: .textForeground)
+            backgroundColor = UIColor.from(scheme: .textBackground)
+
+        case .notClassified:
+            securityLevelLabel.text = "NOT Classified" // TODO: Translation: Need to clarify
```
Comment (David-Henner) [inline Wire-iOS/Sources/UserInterface/UserProfile/Views/SecurityLevelView.swift:51]: same here

File: Wire-iOS/Sources/UserInterface/UserProfile/Views/SecurityLevelView.swift [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+    private func createConstraints() {
+        securityLevelLabel.translatesAutoresizingMaskIntoConstraints = false
+
+        NSLayoutConstraint.activate([
+          securityLevelLabel.topAnchor.constraint(equalTo: topAnchor),
+          securityLevelLabel.centerXAnchor.constraint(equalTo: centerXAnchor),
+          securityLevelLabel.heightAnchor.constraint(equalToConstant: 24),
+          securityLevelLabel.bottomAnchor.constraint(equalTo: bottomAnchor)
+        ])
```
Comment (David-Henner) [inline Wire-iOS/Sources/UserInterface/UserProfile/Views/SecurityLevelView.swift:83]: you can do this in one line with the method `fitIn(view: UIView, insets: UIEdgeInsets = .zero)`

File: Wire-iOS/Sources/UserInterface/UserProfile/Views/SecurityLevelView.swift [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import UIKit
+import WireDataModel
+import WireSyncEngine
+
+final class SecurityLevelView: UIView {
+    let securityLevelLabel = UILabel()
```
Comment (David-Henner) [inline Wire-iOS/Sources/UserInterface/UserProfile/Views/SecurityLevelView.swift:24]:
```suggestion
    private let securityLevelLabel = UILabel()
```

**Review by David-Henner — COMMENTED** [review]
could you add some snapshot tests?

**Pull request merged**